dist/model.js

```diff
@@ -0,0 +1,44 @@
+class TempManager {
+	constructor() {
+		this.cityData = []
+	}
+
+	getSavedCities = async function () {
+		let result = await $.get('/cities')
+
+		for (let city of result) {
```

Comment:
Too many calls to the server. You will call the server for each saved city. Instead, do this on the server side (assuming you want to get new data every time you pull from the DB).